pkg/api/scylla/validation/cluster_validation.go

```diff
@@ -160,7 +117,9 @@ func ValidateAlternatorSpec(alternator *scyllav1.AlternatorSpec, fldPath *field.
 func ValidateScyllaClusterSpec(spec *scyllav1.ScyllaClusterSpec, fldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
 
-	rackNames := sets.NewString()
+	allErrs = append(allErrs, ValidateStructSliceFieldUniqueness(spec.Datacenter.Racks, func(rackSpec scyllav1.RackSpec) string {
```

Comment:
can we have the changes not strictly related to API migration extracted to a separate PR? (common funcs, refactoring etc) It would make it easier to review the validation of the new object

Reply:
discussed offline, resolving